### DISCUSSION TITLE
Can somebody explain how Blackhole works?  (it just routes audio from its input to output?)

### DISCUSSION THREAD

**Discussion by sofakng** [ExistentialAudio/BlackHole#735] [General]
I'm trying to configure macOS to use AU Lab for EQ on my headphones and I'm a bit confused.

Blackhole is a 'loopback audio driver' so does that mean it just sends audio from its input to its output?  (or the other way around?)

Do I understand this correctly?  If I set 'BlackHole 2ch' as my system output device then macOS will send audio to it.  This audio is sent to the BlackHole OUTPUT side.

I can then use another application with the BlackHole INPUT (as a 'microphone' or similar) and that application will receive the audio from BlackHole's OUTPUT?

How does the Multi-output device factor into this?  My goal is to use AU Lab to EQ on the built-in headphone output...

**Comment by devinroth** [comment]
Correct. Output-> Input.

A multi output sends audio to more than one output. Ie: BlackHole + Speakers.

Devin

**Reply by sofakng** [reply]
Thanks very much!
